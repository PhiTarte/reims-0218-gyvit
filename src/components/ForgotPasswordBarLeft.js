import React from "react"
import { Container } from "reactstrap"
import ForgotPasswordBackToSignIn from "./ForgotPasswordBackToSignIn"
import ForgotPasswordTitle from "./ForgotPasswordTitle"
import ForgotPasswordEmail from "./ForgotPasswordEmail"
import ForgotPasswordButton from "./ForgotPasswordButton"
import ForgotPasswordGetStarted from "./ForgotPasswordGetStarted"

const ForgotPasswordBarLeft = () => (
  <Container
    style={{
      "margin-left": "30px",
      "background-color": "#ffffff",
      "margin-right": "10px"
    }}
  >
    <ForgotPasswordBackToSignIn />
    <ForgotPasswordTitle />
    <ForgotPasswordEmail />
    <ForgotPasswordButton />
    <ForgotPasswordGetStarted />
  </Container>
)

export default ForgotPasswordBarLeft
