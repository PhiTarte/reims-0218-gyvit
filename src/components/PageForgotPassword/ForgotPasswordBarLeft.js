import React from "react"
import { Container, Form } from "reactstrap"
import ForgotPasswordBackToSignIn from "./ForgotPasswordBackToSignIn"
import ForgotPasswordTitle from "./ForgotPasswordTitle"
import ForgotPasswordEmail from "./ForgotPasswordEmail"
import ForgotPasswordButton from "./ForgotPasswordButton"
import ForgotPasswordGetStarted from "./ForgotPasswordGetStarted"

const ForgotPasswordBarLeft = () => (
  <Container
    style={{
      marginLeft: "30px",
      backgroundColor: "#ffffff",
      marginRight: "10px"
    }}
  >
    <Form>
      <ForgotPasswordBackToSignIn />
      <ForgotPasswordTitle />
      <ForgotPasswordEmail />
      <ForgotPasswordButton />
      <ForgotPasswordGetStarted />
    </Form>
  </Container>
)

export default ForgotPasswordBarLeft
