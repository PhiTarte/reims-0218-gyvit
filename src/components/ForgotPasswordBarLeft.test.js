//TODO: test:  navbar top with search bar, notification icon and profile.

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import ForgotPasswordBarLeft from "./ForgotPasswordBarLeft"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ForgotPasswordBarLeft />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ForgotPasswordBarLeft />).toJSON()
  expect(tree).toMatchSnapshot()
})
