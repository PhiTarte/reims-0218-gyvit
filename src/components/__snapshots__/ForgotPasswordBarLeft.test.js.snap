// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
  style={
    Object {
      "background-color": "#ffffff",
      "margin-left": "30px",
      "margin-right": "10px",
    }
  }
>
  <p>
    Back to Sign in
  </p>
  <div
    style={
      Object {
        "padding-top": "40%",
      }
    }
  >
    <p
      style={
        Object {
          "color": "#3b425d",
          "font-size": "24px",
          "font-stretch": "normal",
          "font-style": "normal",
          "font-weight": "normal",
          "height": "24px",
          "letter-spacing": "normal",
          "line-height": "normal",
          "text-align": "left",
        }
      }
    >
      Forgot your password ?
    </p>
    <p
      style={
        Object {
          "color": "#9da5c9",
          "font-size": "18px",
          "font-stretch": "normal",
          "font-style": "normal",
          "font-weight": "normal",
          "letter-spacing": "normal",
          "line-height": "normal",
          "margin-top": "15px",
          "text-align": "left",
        }
      }
    >
      Enter your email address below and we'll get you back on track.
       
    </p>
  </div>
  <form
    className=""
    style={
      Object {
        "margin-bottom": "40px",
        "margin-top": "60px",
      }
    }
  >
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="email"
      >
        Email 
      </label>
      <input
        className="form-control"
        id="IdEmail"
        name="email"
        placeholder="Email"
        type="email"
      />
    </div>
  </form>
  <div>
    <button
      className="btn btn-info"
      onClick={[Function]}
      style={
        Object {
          "background-image": "linear-gradient(to left, #57aad1, #7a57d1)",
          "border-radius": "100px",
          "height": "54px",
          "width": "192px",
        }
      }
      type={undefined}
    >
      Request reset link
    </button>
  </div>
  <div
    style={
      Object {
        "padding-top": "20%",
      }
    }
  >
    <p>
      Don't have an account ? Get started
    </p>
  </div>
</div>
`;
